frontend/src/component/project/Project/PaginatedProjectFeatureToggles/FeatureToggleCell/FeatureToggleCell.tsx

```diff
@@ -90,3 +96,4 @@ export const PlaceholderFeatureToggleCell = () => (
         <div data-loading>toggle</div>
     </StyledSwitchContainer>
 );
+export const ArchivedFeatureToggleCell = () => <StyledDiv>-</StyledDiv>;
```

Comment:
Wonder if we should do `aria-hidden=true` here. It's not very helpful for screen readers if it reads out "development: hyphen, production: hyphen", etc. (which I *suspect* it will). Either that, or use ScreenReaderOnly to indicate that it has been disabled? But then, you'd get that for every row, which is also probably not super helpful. Hmm 🤔  What do you think?

Reply:
I think aria hidden makes sense.